cube/src/main/java/org/jboss/arquillian/ce/cube/CEEnvironmentProcessor.java

```diff
@@ -142,18 +144,24 @@ private void registerRoutes(CECubeConfiguration configuration, OpenShiftClient c
     }
 
     private void processTemplate(TestClass tc, OpenShiftAdapter client, CECubeConfiguration configuration)
-        throws DeploymentException {
+            throws DeploymentException {
         final StringResolver resolver = Strings.createStringResolver(configuration.getProperties());
         final Template template = ReflectionUtils.findAnnotation(tc.getJavaClass(), Template.class);
         final String templateURL = readTemplateUrl(template, configuration, resolver);
         final List<? extends OpenShiftResource> resources;
+
+        if (templateURL == null || templateURL.length() == 0) {
```

Comment:
@alesj the only objective of this PR is continue with the tests if no template is found.
If the template is not provided as annotation or via System Properties the tests will proceed normally only with the runinpod and the cleanup will also work fine after the tests is complete.
The integration tests does not use the template, and doing it we save much time running the integration tests.